config/.rubocop.yml

```diff
@@ -1,8 +1,11 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
This file has to be in the root of the repo.

Once you move it, I think travis will be green. 😄 

`./.rubocop.yml`